src/screens/TestCases/ReviewCommentMaster/ReviewCommentMasterComponent.js

```diff
@@ -1,133 +1,120 @@
-import React, { useEffect, useState } from "react";
-import DataTable from "react-data-table-component";
-import PageHeader from "../../../components/Common/PageHeader";
-import { ExportToExcel } from "../../../components/Utilities/Table/ExportToExcel";
-import ReviewCommentMasterModal from "./Validation/ReviewCommentMasterModal";
-import { useDispatch, useSelector } from "react-redux";
-import { getReviewCommentMasterListThunk } from "../../../redux/services/testCases/reviewCommentMaster";
-import { customSearchHandler } from "../../../utils/customFunction";
-import TableLoadingSkelton from "../../../components/custom/loader/TableLoadingSkelton";
-import { Col, Row } from "react-bootstrap";
+import React, { useEffect, useState } from 'react';
+import DataTable from 'react-data-table-component';
+import PageHeader from '../../../components/Common/PageHeader';
+import { ExportToExcel } from '../../../components/Utilities/Table/ExportDataFile';
+import ReviewCommentMasterModal from './Validation/ReviewCommentMasterModal';
+import { useDispatch, useSelector } from 'react-redux';
+import { getReviewCommentMasterListThunk } from '../../../redux/services/testCases/reviewCommentMaster';
+import { customSearchHandler } from '../../../utils/customFunction';
+import TableLoadingSkelton from '../../../components/custom/loader/TableLoadingSkelton';
+import { Col, Row } from 'react-bootstrap';
 
 function ReviewCommentMasterComponent() {
   const dispatch = useDispatch();
 
   // // redux state
-  const { reviewCommentMasterList, isLoading } = useSelector(
-    (state) => state?.reviewCommentMaster
-  );
+  const { reviewCommentMasterList, isLoading } = useSelector((state) => state?.reviewCommentMaster);
 
-  const [searchValue, setSearchValue] = useState("");
-  const [filteredReviewCommentMasterList, setFilteredReviewCommentMasterList] =
-    useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [filteredReviewCommentMasterList, setFilteredReviewCommentMasterList] = useState([]);
 
   const [addEditReviewCommentModal, setAddEditReviewCommentModal] = useState({
-    type: "",
-    data: "",
-    open: false,
+    type: '',
```

Comment:
if you use type then modal name should be according it

Reply:
resolved